### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by mitdesai** [kubernetes/dns#654] [Q&A]
When can we expect a new release tag, now that PR #649 is merged?

**Comment by cwayne18** [comment]
I have the same question, and I'd be happy to help in any way to get this sorted asap

**Comment by DamianSawicki** [comment]
A new tag is cut now https://github.com/kubernetes/dns/releases/tag/1.24.0, and we are in the process of image promotion (https://github.com/kubernetes/k8s.io/pull/7607, https://github.com/kubernetes/kubernetes/pull/129175).

**Comment by dereknola** [comment]
Bump, we should get all the dependabot updates that have merged in the last month into a new release.

**Reply by Michcioperz** [reply]
I believe Damian has cut a 1.26.5 as of either today or last Friday, it just doesn't show on the main repo page

**Reply by DamianSawicki** [reply]
Correct, I cut https://github.com/kubernetes/dns/releases/tag/1.26.5 actually on Saturday morning. However, I haven't completed yet further steps of the release process, so the reminder makes perfect sense :) I'll try to promote images right now.

**Reply by DamianSawicki** [reply]
> it just doesn't show on the main repo page

Yeah, the main page shows only the *latest release*, which is 1.26.0, and it doesn't show all the 1.26.1–1.26.5 tags.

> I'll try to promote images right now.

https://github.com/kubernetes/k8s.io/pull/8438

**Reply by dereknola** [reply]
Sorry I didn't see the tags, great work @DamianSawicki, appreciate it.
